### PR TITLE
some bugs fixed and created mobile version of main page

### PR DESCRIPTION
**Pull request by Rozygeldi94**

### CODE REVIEW

File: src/components/Carousel/Carousel.tsx [hunk 1]
```diff
@@ -1,176 +1,231 @@
...
-  const [isFirstSlick, setIsFirstSlick] = useState(true);
-  const [isLastSlick, setIsLastSlick] = useState(false);
-  const containerRef = useRef<HTMLDivElement>(null);
-
-  const handlePrevClick = () => {
-    setPrevClick(true);
-    ref.current?.prev();
-    setTimeout(() => setPrevClick(false), 0); // Reset click state after 0s
-  };
+  const responsiveSettings = [
```
Comment (mskmee) [inline src/components/Carousel/Carousel.tsx:14]: This constant should be in a separate constants file or outside the component function. The first approach is more flexible

File: src/components/Carousel/Carousel.tsx [hunk 2]
```diff
@@ -1,176 +1,231 @@
...
-      </Carousel>
-      <SliderButtons
-        handleNextClick={handleNextClick}
-        handlePrevClick={handlePrevClick}
-        isFirstSlick={isFirstSlick}
-        isLastSlick={isLastSlick}
-      />
+        <div
+          className={styles.track}
+          style={{
```
Comment (mskmee) [inline src/components/Carousel/Carousel.tsx:163]: why did you use style here? Could you use className in this case?

File: src/components/SingleProduct/index.tsx [hunk 3]
```diff
@@ -118,7 +118,7 @@ export const Product: FC<IProductProps> = ({ reviewsLength }) => {
   };
 
   return (
-    <section className={style['product']} id="product">
+    <section className={classNames(style['product'])} id="product">
```
Comment (mskmee) [inline src/components/SingleProduct/index.tsx:121]: classnames is unnecessary here.

File: src/components/SliderNav/SliderNav.tsx [hunk 4]
```diff
@@ -22,7 +23,7 @@ export const SliderNav: FC<ISliderNavProps> = ({
     setIsClicked((prevState) => !prevState);
   };
   return (
-    <div className={styles.sliderText}>
+    <div className={classNames(styles.sliderText)}>
```
Comment (mskmee) [inline src/components/SliderNav/SliderNav.tsx:26]: classnames is unnecessary here.

File: src/components/benefitsList/benefits.tsx [hunk 5]
```diff
@@ -17,7 +18,7 @@ const Benefit: FC<BenefitItem> = ({ imageUri, text, imageAlt }) => {
 
 const Benefits = () => {
   return (
-    <section className={styles.wrapper}>
+    <section className={classNames(styles.wrapper)}>
```
Comment (mskmee) [inline src/components/benefitsList/benefits.tsx:21]: classnames is unnecessary here.

File: src/hooks/useIsFixedHeader.ts [hunk 6]
```diff
@@ -9,6 +9,8 @@ export const useIsFixedHeader = () => {
 
     const handleScroll = () => {
       if (headerTopSectionHeight) {
+        console.log(headerTopSectionHeight);
```
Comment (mskmee) [inline src/hooks/useIsFixedHeader.ts:12]: please remove debug logs in PRs

**Review by mskmee — CHANGES_REQUESTED** [review]

File: src/components/Carousel/Carousel.tsx (hunk 2 above)
Comment (Rozygeldi94) [reply src/components/Carousel/Carousel.tsx:163]: here is dynamic state (currentTranslate) that's why

**Pull request closed without merging**